src/middlewared/middlewared/etc_files/local/netdata/netdata.conf

```diff
@@ -0,0 +1,55 @@
+<%
+	# We do an empty update to ensure that additional_params value is filled when we get netdata global settings
+	netdata_global_settings = middleware.call_sync('netdata.update', {})
```

Comment:
That is super weird. Why would additional_params not be filled?

Reply:
Because when the model is created, neither does it has any alarms configured in the db nor the additional param field value. An empty update takes care of both of these and sets up the db for this by reading the relevant data from files

Reply:
We should probably create it in the migration, then? To avoid updating the service again everytime its generated?

Reply:
When the build is being created, aren't migrations applied then ? I am not sure if those files would be there at that point ?